youtube_dl/extractor/tele5.py

```diff
@@ -0,0 +1,40 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from .nexx import NexxIE
+
+
+class Tele5IE(InfoExtractor):
+    _VALID_URL = r'https://www.tele5.de/[mediathek/filme-online/videos|tv/]'
+
+    _TESTS = [{
+        'url': 'https://www.tele5.de/mediathek/filme-online/videos?vid=1550589',
+        'info_dict': {
+            'id': '1550589',
+            'ext': 'mp4',
+            'upload_date': '20180822',
+            'timestamp': 1534927316,
+            'title': 'SchleFaZ: Atomic Shark'
+        }
+    }, {
+        'url': 'https://www.tele5.de/tv/dark-matter/videos',
+        'info_dict': {
+            'id': '1548206',
+            'ext': 'mp4',
+            'title': 'Folge Sechsundzwanzig',
+            'timestamp': 1533664358,
+            'upload_date': '20180807'
+        }
```

Comment:
I've already pointed out: **remove all duplicate tests**. Or make them only_matching.

Reply:
Why is this a duplicate test although it tests another URL pattern?

Reply:
Because all of them use the same extraction scenario.